controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -53,7 +77,7 @@ wfa_ca::eWfaCaCommand wfa_ca::wfa_ca_command_from_string(std::string command)
 void wfa_ca::reply_invalid_error(Socket *sd, ieee1905_1::CmduMessageTx &cmdu_tx,
                                  eWfaCaStatus status, const std::string &description)
 {
-    if (status != eWfaCaStatus::INVALID || status != eWfaCaStatus::ERROR) {
+    if (status != eWfaCaStatus::INVALID && status != eWfaCaStatus::ERROR) {
```

Comment:
I didn't think it was a significant change for a separate commit.
And since this module is currently in development, it felt natural to me to push it as part of this commit.

Reply:
still, take this as an area for improvement for next time - splitting to commits especially if they are not related to the actual change helps for a better code review process. You should keep in mind that code review is essential and benefitial especially to the one who is doing the change, so thriving for a clear and well split commit is a requirement, not a recommendation.